dora/src/typeck/lookup.rs

```diff
@@ -292,7 +292,7 @@ impl<'a, 'ast> MethodLookup<'a, 'ast> {
                 let list_id = self.vm.lists.lock().insert(cls_tps);
                 BuiltinType::Class(cls_id, list_id)
             }
-
+            LookupKind::Method(BuiltinType::Module(mid)) => BuiltinType::Module(mid),
```

Comment:
This would get me through type checking, but then it fails later in codegen:

```
  21: dora::semck::specialize::replace_type_param
             at dora/src/semck/specialize.rs:411
  22: dora::baseline::codegen::AstCodeGen::specialize_type
             at dora/src/baseline/codegen.rs:3474
  23: dora::baseline::codegen::AstCodeGen::determine_call_args_and_types
             at dora/src/baseline/codegen.rs:3617
  24: dora::baseline::codegen::AstCodeGen::build_call_site
             at dora/src/baseline/codegen.rs:3535
  25: dora::baseline::codegen::AstCodeGen::build_call_site_id
             at dora/src/baseline/codegen.rs:3563
  26: dora::baseline::codegen::AstCodeGen::emit_call
             at dora/src/baseline/codegen.rs:2026
  27: dora::baseline::codegen::AstCodeGen::emit_expr
             at dora/src/baseline/codegen.rs:718
  28: dora::baseline::codegen::AstCodeGen::emit_expr_result_reg
             at dora/src/baseline/codegen.rs:692
  29: dora::baseline::codegen::AstCodeGen::emit_stmt_expr
             at dora/src/baseline/codegen.rs:582
```

Reply:
Ok, this change is wrong anyway.